package-build/tools/checkEnvironment.ps1

```diff
@@ -0,0 +1,23 @@
+$ErrorActionPreference = 'Stop';
+
+Write-Host 'Checking environment...'
+
+# Check system PATH length
+# https://support.microsoft.com/en-ca/help/830473/command-prompt-cmd-exe-command-line-string-limitation
+$PathLen = $Env:PATH.Length
+$MaxPathLen = 8191
+if ($PathLen -gt $MaxPathLen) {
+    Write-Warning "The length of %PATH% is over $MaxPathLen."
+    Write-Warning "Reduce the %PATH% length and retry again."
```

Comment:
Good, but Maybe more explanation since many customers might not understand the implication.
"Windows has a path length limitation. Your system path is quite long, which prevents ROS from installing or starting correctly.  Please reduce the size of your system path though the Windows System Advanced Control panel"